assignment.hs

```diff
@@ -23,7 +23,7 @@ import Prelude
 -- returns the reciprocal of that number. This should make (1a) and (1b) 
 -- correct in the main function at the end
 
-reciprocal x = . . .
+reciprocal x = (1/x)
```

Comment:
It's not an error, but the parentheses are not needed. This is a subtle point, but I feel that leaving them out actually makes the code easier to read: when I first looked at this I spent a second or two thinking "why do we need parens here?"